ynput/OpenPype
# Unreal: 🚚 move Unreal plugin to separate repository

## Changelog Description
To support Epic Marketplace have to move AYON Unreal integration plugins to separate repository. This is replacing current files with git submodule, so the change should be functionally without impact.

New repository lives here: https://github.com/ynput/ayon-unreal-plugin

## Testing notes:
1. Make sure you have the submodule initialized and checked out
2. Try to bootstrap Unreal project from clean state (delete AYON plugin from Engine, start with the empty task)
